machine-templates/interoutevdc/0/uiUrl

```diff
@@ -0,0 +1 @@
+https://raw.githubusercontent.com/Interoute/ui-driver-interoutevdc/v1.0/dist/component.js
```

Comment:
This doesn't work... github returns `Content-Type: text/plain` and browsers will refuse to interpret as JS.  Also only the domain that the driver is loaded from is automatically whitelisted, so the user would need to add `raw.githubusercontent.com` to the `api.proxy.whitelist` to allow it to be loaded.